templates/rhdm77-authoring-ha.yaml

```diff
@@ -1114,7 +1114,7 @@ objects:
           - name: KIE_SERVER_ROUTE_NAME
             value: "insecure-${APPLICATION_NAME}-kieserver"
           - name: KIE_SERVER_STARTUP_STRATEGY
-            value: "ControllerBasedStartupStrategy"
+            value: "OpenShiftStartupStrategy"
```

Comment:
The file templates/docs/rhdm77-authoring-ha.adoc should be updated accordingly to state "KIE_SERVER_STARTUP_STRATEGY" parameter from "ControllerBasedStartupStrategy" to "OpenShiftStartupStrategy".